libs/go/zmscli/cli.go

```diff
@@ -14,11 +14,20 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/ardielle/ardielle-go/rdl"
 	"github.com/AthenZ/athenz/clients/go/zms"
+	"github.com/ardielle/ardielle-go/rdl"
 	"golang.org/x/net/proxy"
 )
 
+const (
+	// JSONOutputFormat is the JSON output format for commands.
+	JSONOutputFormat = "json"
```

Comment:
Goodjob having a library to take care of the json output. You'll notice that for yaml the prints are still hardcoded (with spaces for indentations).
Is it possible for you to replace the current yaml printing code with a library that will print the yaml output, similar to how you did in Json?

Reply:
So the formatting for JSON works because the APIs return JSON responses, so it's a pretty simple thing to pretty-print those.
But I'm afraid it wouldn't be possible to do so for the YAML output since there are no YAML responses from the APIs, which is probably why they've been hardcoded.

Reply:
Yeah the API returns it as Json but I thought about having a library that converts from Json to Yaml.
That way adding new commands or changing existing commands will not require fumbling with Yaml indentations.
In any case this can be taken care of in a future PR

Reply:
Oh in that way, https://github.com/icza/dyno is one such library that can do that. A separate PR would be the way to go, you're right.

Reply:
https://github.com/ghodss/yaml also looks good

Reply:
Yes, that'll do it.